### PR TITLE
chore(Routes): modularize routes and protect sidekiq

### LINKED ISSUE #107: Protect sidekiq dashboard endpoint
The endpoint is currently public. Should update it with some authentication layer

### PR DESCRIPTION
**Pull request by mapra99**
Closes #107
A couple changes:
- Sidekiq dashboard was totally public, now it will be protected by HTTP user auth.
- Routes file is split into separated files to make it a bit cleaner

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,36 +2,8 @@
...
-      namespace :auth do
-        post 'signup'
-        post 'login'
-        post 'confirmation'
-        post 'logout'
-      end
-      resources :sessions, only: %i[create], param: :uuid
-    end
-  end
+  # draw(:sidekiq)
```
Comment (mapra99) [inline config/routes.rb]:
```suggestion
```

Comment (ImMPrada) [inline config/routes.rb]:
```suggestion
 
```

**Review by ImMPrada — APPROVED** [review]
UFF NICE!

File: config/routes.rb (hunk 1 above)
Comment (mapra99) [reply config/routes.rb]: good catch! change made

**Pull request merged**